pom.xml

```diff
@@ -103,7 +190,7 @@
         <plugin>
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-compiler-plugin</artifactId>
-          <version>3.9.0</version>
```

Comment:
3.10.0 breaks on windows.

Reply:
Oh, I wasn't aware of that, I'll downgrade it again